platform/commonUI/edit/src/actions/CancelAction.js

```diff
@@ -82,7 +66,7 @@ define(
         CancelAction.appliesTo = function (context) {
             var domainObject = (context || {}).domainObject;
             return domainObject !== undefined &&
-                domainObject.hasCapability("editor");
+                domainObject.getCapability("status").get("editing");
```

Comment:
@VWoeltjen in your mind, is there a semantic difference between `isEditingContextRoot()` and the function that I created `inEditingContext()`. If no, I'll rename it.